### PR TITLE
Add Windows support

### PR DESCRIPTION
**Pull request by euclio**
This pull request will add Windows support to the plugin, once suan/instant-markdown-d#15 is merged.

### CODE REVIEW

**Comment by suan** [comment]
@euclio I know it's been _years_, but if you're still there, would you mind resolving conflicts with the latest `master`? I've verified your change on both repos works on my Windows VM

File: after/ftplugin/markdown/instant-markdown.vim [hunk 1]
```diff
@@ -17,14 +17,44 @@ function! s:system(cmd, stdin)
...
+    " system(). Instead, the lines are saved to a file and then piped into the
+    " command.
+    if len(a:stdinLines)
+        let tmpfile = tempname()
+        call writefile(a:stdinLines, tmpfile)
+        let command = 'type ' . tmpfile . ' | ' . a:cmd
+    else
+        let command = a:cmd
+    endif
+    exec 'silent !start /b cmd /c ' . command . ' > NUL'
```
Comment (suan) [inline after/ftplugin/markdown/instant-markdown.vim]: Why can't we use `call s:system(...)` like the Unix case?

Comment (suan) [inline after/ftplugin/markdown/instant-markdown.vim]: Ah - is it cause the ordering and where the pipe is is different when using the `!start` command in Windows?

**Comment by euclio** [comment]
@suan, it has been a while! Rebased.

**Pull request closed without merging**